### PR TITLE
Monkeypox HL7 ELR Message Handling

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR ...

Test Steps:
1. *Include steps to test these changes*

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: prime-router/metadata/schemas/monkeypox.schema [hunk 1]
```diff
@@ -0,0 +1,994 @@
+---
+# This is set of standard data elements for covid-19 topic
```
[the remark this answers is not in the document]
Comment (MauriceReeves-usds) [reply prime-router/metadata/schemas/monkeypox.schema:2]: Yes, good catch, thank you

File: prime-router/metadata/valuesets/monkeypox.valuesets [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+---
+- name: monkeypox/specimen_type
+  system: SNOMED_CT
+  version: 2.67
+  referenceUrl: ""
+  values:
+    - code: "257261003"
+      display: "Swab (specimen)"
+
+- name: monkeypox/test_result
```
Comment (jimduff-usds) [inline prime-router/metadata/valuesets/monkeypox.valuesets:11]: OK for now.   

Still need these test_result codes:



Inconclusive | 419984006
Equivocal | 42425007
Not detected | 260415000
Test not done | 373121007

Also need these LOINC test_performed codes:



Test | Test Performed   LOINC Long Name | LOINC Code
-- | -- | --
Orthopoxvirus | Orthopoxvirus DNA [Presence] in Specimen by NAA with probe detection | 41853-3
Non-variola Orthopoxvirus | Orthopoxvirus non-variola DNA [Presence] in Specimen by NAA with probe detection | 100434-0
Monkeypox Virus | Monkeypox virus DNA [Presence] in Specimen by NAA with probe detection | 100383-9

**Review by jimduff-usds — APPROVED** [review]
I have made a few tweaks, and I have tested both old covid data and new monkeypox data.  I think we can do the Staging and Prod settings creation with no PR needed.

File: prime-router/settings/organizations.yml [hunk 3]
```diff
@@ -1343,7 +1367,37 @@
...
+      topic: monkeypox
+      routingFilter:
+        - "AllowAll()"
+      processingModeFilter:
+        - "AllowAll()"
+      qualityFilter:
+        - "AllowAll()"
+      customerStatus: active
+      jurisdictionalFilter:
+        - orEquals(ordering_facility_state, MN, patient_state, MN)
```
[the remark this answers is not in the document]
Comment (MauriceReeves-usds) [reply prime-router/settings/organizations.yml:1392]: @PatriciaPerozoUSDS the organizations file doesn't update production settings. It's just for local dev environments.

**Pull request merged**